cla_backend/apps/knowledgebase/tests/test_csv_2_fixture.py

```diff
@@ -12,9 +15,25 @@ class TestCSV2Fixture(TestCase):
     def setUp(self):
         self.datetime_now = datetime.now().replace(tzinfo=timezone.get_current_timezone()).isoformat()
 
-    def calculate_pk_range(self, article_category_matrices):
-        min_range = min(article_category_matrices, key=lambda x: x["pk"])
-        max_range = max(article_category_matrices, key=lambda x: x["pk"])
```

Comment:
I changed the arg name from article_category_matrices to article_category_matrix

Reply:
Why? Are you only passing it one matrix object now? I don't think that's the case...

Reply:
Yes it only takes in 1 article_category_matrix. It was called matrices before because I wanted to use the same name it was called/references to in other files.

This is evidenced by my newest comment below